data/json/items/appliances.json

```diff
@@ -71,5 +71,39 @@
     "price_postapoc": 5000,
     "symbol": "L",
     "color": "white"
+  },
+  {
+    "type": "GENERIC",
+    "id": "static_solar_panel",
+    "name": { "str": "static solar panel" },
+    "description": "A set of photovoltaic power generators ready to be installed to a power grid, providing electricity from solar radiation.",
+    "weight": "12 kg",
+    "to_hit": -1,
+    "material": [ "glass", "steel" ],
+    "volume": "10 L",
+    "longest_side": "120 cm",
+    "bashing": 4,
+    "price": 10000,
+    "price_postapoc": 2000,
+    "symbol": "#",
+    "color": "light_blue"
+  },
+  {
+    "type": "GENERIC",
+    "id": "grid_battery_small",
+    "name": { "str": "Small power grid battery" },
```

Comment:
name": { "str": "Small power grid battery", "str_pl": "Small power grid batteries" },

Reply:
I was thinking of changing this to something like 'small appliance battery' anyhow since I don't know if 'power grid' is the proper name for the appliance/furniture system. I want to settle on a naming schema I can keep consistent as I add more stuff.